### ISSUE TITLE
Collaborate on asyncp

### ISSUE THREAD

**Issue by jgornick** [toniov/p-iteration#2]
Hi @toniov! Great stuff with p-iteration! I wanted to share a project I started a while back which is a port of the async.js library to use native promises. All of the methods you've implemented in p-iteration are also available in asyncp. It would be nice to collaborate and work on asyncp together.

[jgornick/asyncp](https://github.com/jgornick/asyncp)

Let me know what you think. Thanks!

**Comment by toniov** [comment]
Hi there!

I took a look to `asyncp` and looks cool, I guess it took you some time and effort to adapt that many methods of async.

I wouldn't mind collaborating there as well when I have some spare time! But I'll continue maintaining and improving `p-iteration`, they serve similar purposes but they are different in some ways.

For example, `p-iteration` keeps the same syntax as the ES5 Array iteration methods, and only includes these basic methods, being able to write asynchronous code in the same way as we are used to do synchronously; that's why I decided to create this module instead of using an existent one.
